src/main/kotlin/uk/gov/justice/hmpps/offenderevents/subscribe/PrisonerEventsListener.kt

```diff
@@ -46,6 +47,7 @@ class PrisonerEventsListener(
           SendMessageRequest.builder()
             .queueUrl(attributes.queueUrl)
             .messageBody(message)
+            .messageAttributes(mapOf("eventType" to MessageAttributeValue.builder().dataType("String").stringValue(eventType).build()))
             .delaySeconds(delay.toSeconds().toInt())
```

Comment:
way over the top I know; but maybe add a function in library:
`fun eventType(eventType: String) = mapOf("eventType" to MessageAttributeValue.builder().dataType("String").stringValue(eventType).build())`

To get rid of all this builder code

Reply:
as I was writing the code for the fourth time it did cross my mind that I should do just that, yes, will pick up next.